README.md

```diff
@@ -29,41 +29,51 @@ conda activate aspire_env
 pip install aspire
 ```
 
-The final step above should install any dependent packages from `pip` automatically.
+The final step above should install any dependent packages from `pip` automatically. To see what packages are required, browse `setup.py`.
```

Comment:
Is there no way to do this from the command line? I tried `python setup.py --requires`, but it wasn't too informative…

Reply:
There are a few ways.. I put one in the readme, bc1a86c1edb495341378f5e92122150f9372eafc . As follows:

```
(base) ➜  ASPIRE-Python git:(resolve_req_192) ✗ pip show aspire           
Name: aspire
Version: 0.5.28
Summary: Algorithms for Single Particle Reconstruction
Home-page: https://github.com/ComputationalCryoEM/ASPIRE-Python
Author: Joakim Anden, Yoel Shkolnisky, Itay Sason, Robbie Brook, Vineet Bansal, Junchao Xia
Author-email: devs.aspire@gmail.com
License: GPLv3
Location: /scratch/Dropbox/PU/aspire/ASPIRE-Python/src
Requires: click, finufftpy, importlib-resources, joblib, jupyter, matplotlib, mrcfile, numpy, numpydoc, pandas, pyfftw, python-box, pytest, pytest-cov, python-box, scipy, scikit-learn, scikit-image, sphinxcontrib-bibtex, sphinx-rtd-theme, tqdm
Required-by: 
```

Reply:
It seems that this only works if the package is installed. Is there a way to list them without installing the package. Otherwise, let's just revert to the previous formulation.

Reply:
Interesting, I didn't realize. That is a useless command.  Other solutions seem to involve installing other packages. Its probably fine to expect people either read the file or use google here. Reverted.